THCudaCheck FAIL file=/pytorch/aten/src/THC/THCGeneral.cpp line=663 error=11 : invalid argument

700 -------------------------------------------------
Loading restarting config from: /home/monica/IIC/datasets/iid_private/700/config.pickle
avg_pool_sz 7
semisup: False
(_sobel_multioutput_make_transforms) config.include_rgb: False
not using cutout
not demeaning data
not per image demeaning data
at batch 0
at batch 100
at batch 200
at batch 300
at batch 400
at batch 500
[-35.48533012 -35.0894176  -35.5900284  -34.79831956 -34.6737912 ]
best sub_head by loss: 2
Traceback (most recent call last):
  File "/home/monica/anaconda3/envs/py275/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/home/monica/anaconda3/envs/py275/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/monica/IIC/code/scripts/cluster/analysis/print_sub_heads_eval.py", line 55, in <module>
    main()
  File "/home/monica/IIC/code/scripts/cluster/analysis/print_sub_heads_eval.py", line 53, in main
    compare=True)
  File "code/utils/cluster/cluster_eval.py", line 301, in get_subhead_using_loss
    best_epoch = np.argmax(np.array(config.epoch_acc))
  File "/home/monica/anaconda3/envs/py275/lib/python2.7/site-packages/numpy/core/fromnumeric.py", line 1037, in argmax
    return _wrapfunc(a, 'argmax', axis=axis, out=out)
  File "/home/monica/anaconda3/envs/py275/lib/python2.7/site-packages/numpy/core/fromnumeric.py", line 51, in _wrapfunc
    return getattr(obj, method)(*args, **kwds)
ValueError: attempt to get argmax of an empty sequence
